readme.md

```diff
@@ -0,0 +1,108 @@
+The main purpose of this repository is to provide you with sample Grafana dashboard to monitor you ODP stack. Dashboard is focusing on main metrics that allow you to verify that your ODP stack is running properly and is in good state.
+
+# Enable metrics for Data Broker and Data Connect
+
+## Data Connect
+In order to enable Data Connect metrics, you need to edit config file (`connect.yaml`).
+
+First of all you need to make sure that your Data Connect config contains these entries:
+```yaml
+server:
+  address: 0.0.0.0
+  port: 9070
+```
+This will enable Spring Boot server and allow you to publish Prometheus endpoint with metrics.
+If port 9070 is not available, it can be changed to other available value.
+
+Next you need to add these entries:
+```yaml
+management:
+  endpoints:
+    web:
+      exposure:
+        include: prometheus
+```
+This will enable Prometheus endpoint with Data Connect, JVM and other metrics. Once enabled, metrics will be available via: `http://[HOST]:[PORT]/actuator/prometheus` where:
+- `HOST` - `server.address` value from `config.yaml`
+- `PORT` - `server.port` value from `config.yaml`
+In our case it will be `http://localhost:9070/actuator/prometheus`.
+
+Data Connect metrics always start with `odp_server_*`.
+
+## Data Broker
+To enable Data Broker metrics, we need to modify `PARMLIB` member and add these parameters:
+```                                  
+KAY.CIDB.STATS=ON
+```
+This will enable stats subsystem. By default, data is being sent every 30 seconds. To change the interval, you can add an entry to overwrite the default value:
+```
+KAY.CIDB.STATS.INTERVAL=60
+```
+This will set refresh interval to 60 seconds.
+
+```
+KAY.CIDB.FWD.ST.SOURCE_STORE=STATS                      
+KAY.CIDB.FWD.ST.SINK_HOST=[DATA_CONNECT_HOST]   
+KAY.CIDB.FWD.ST.SINK_PORT=[DATA_CONNECT_PORT]
+```
+This will create a forwarder and point stats data to your Data Connect instance defined by `SINK_HOST` and `SINK_PORT` parameters.
+Once all that is done, restart Data Broker for new settings to take effect.
+
+At that point, both Data Connect and Data Broker metrics (`odp_server_*` and `odp_broker_*`) will be available on your Prometheus endpoint.
+
+# Installation & Configuration
+Clone the project repository.
+
+There are two ways of how you can you use provided dashboard:
+- in existing Prometheus/Grafana environment
+- in dedicated docker based environment
```

Comment:
Nit: in prose (as opposed to, for example, commands), when referring to the Docker platform, refer to "Docker" with a capital D, not "docker".